app/Http/Resources/EntityPermissionResource.php

```diff
@@ -14,11 +14,13 @@ class EntityPermissionResource extends EntityChild
      */
     public function toArray($request)
     {
-        return $this->entity([
+        return [
             'role' => $this->campaign_role_id,
             'user' => $this->user_id,
```

Comment:
Also to make it clearer for users, please use `user_id` and `campaign_role_id`. The closer the API is to the db, the easier it is for us to debug